qcp/QMatrices.py

```diff
@@ -0,0 +1,42 @@
+class QMatrices:
```

Comment:
If the class just represents a matrix, why not just call it `Matrix` not `QMatrices`?